scripts/google/covid_mobility/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "data",
+            "curator_emails": [
+                "edumorales@google.com"
+            ],
+            "provenance_url": "https://www.google.com/covid19/mobility",
```

Comment:
Could we also add a "provenance_description"? It is a required field in our design.